### DISCUSSION TITLE
What's the purpose of support keys?

### DISCUSSION THREAD

**Discussion by GEOEGII555** [sandboxie-plus/Sandboxie#2740] [Documents and notes]
# What's the purpose of support keys if Sandboxie is an open source program?

### If anyone can easily find code which checks these keys and change it so it would always return that the key is valid, why anyone would buy the supporter keys? (Unless someone wants to support this project of course)

**Comment by isaak654** [comment]
Here is a list of options if you want to compile the project yourself: https://github.com/sandboxie-plus/Sandboxie/issues/1388#issuecomment-1153802583

Note:
- the EV Code Signing Certificate required in option 3 can only be issued for organizations, not individuals
- option 2 requires the activation of the Windows test mode: https://github.com/sandboxie-plus/Sandboxie/issues/1086#issuecomment-894835074

However, you can still get a contributor license for free in exchange of valid contributions: [CONTRIBUTING.md#non-monetary-contributions](https://github.com/sandboxie-plus/Sandboxie/blob/master/CONTRIBUTING.md#non-monetary-contributions)

Anyone who wants to pay to support the project is entitled to do so, unless you're in a business/educational context and need to provide a fixed amount of supporter certificates.

**Comment by ghost** [comment]
Purchasing a license takes few minutes. "Cracking" Sandboxie which is open source software require so much time it is not worth doing (unless you love experiments) and want to learn something new
